core/common/src/main/java/alluxio/metrics/Metric.java

```diff
@@ -207,6 +211,23 @@ public static String getMetricNameWithTags(String name, String... tags) {
     return sb.toString();
   }
 
+  /**
+   * Gets a metric name with a specific user tag.
+   *
+   * @param metricName the name of the metric
+   * @param userName the user
+   * @return a metric name with the user tagged
+   */
+  public static String getMetricNameWithUserTag(String metricName, String userName) {
+    UserKey k = new UserKey(metricName, userName);
+    String result = CACHED_METRICS.get(k);
```

Comment:
is `get` cheaper than `computeIfAbsent` when the key is present? Otherwise the lines calling `get` and checking for `null` aren't needed since `computeIfAbsent` returns the pre-existing value if it was present.

Reply:
Right, get is much cheaper.